afs/afs-storage-api/src/main/java/com/powsybl/afs/storage/AppStorage.java

```diff
@@ -45,13 +45,17 @@
 
     boolean isWritable(String nodeId);
 
+    boolean isEnable(String nodeId);
```

Comment:
You should put it as default method with a throw in impl to allow smooth transition